src/Console/Command/ReplaceFunctionCalls.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+namespace Roave\FunctionFQNReplacer\Console\Command;
+
+use BetterReflection\SourceLocator\Type\DirectoriesSourceLocator;
+use BetterReflection\SourceLocator\Type\PhpInternalSourceLocator;
+use PhpParser\NodeTraverser;
+use PhpParser\ParserFactory;
+use PhpParser\PrettyPrinter\Standard;
+use Roave\FunctionFQNReplacer\Iterator\PhpFilesInDirectoryIterator;
+use Roave\FunctionFQNReplacer\Lookup\FunctionExists;
+use Roave\FunctionFQNReplacer\NodeVisitor\ReplaceUnqualifiedFunctionCallsWithQualifiedReferences;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+final class ReplaceFunctionCalls extends Command
```

Comment:
This is untested, but YOLO. Given that the https://github.com/nilportugues/php-backslasher is tested and stable, I suggest using that for now.